src/category_theory/limits/shapes/equalizers.lean

```diff
@@ -62,13 +63,21 @@ instance fintype_walking_parallel_pair : fintype walking_parallel_pair :=
 open walking_parallel_pair
 
 /-- The type family of morphisms for the diagram indexing a (co)equalizer. -/
-inductive walking_parallel_pair_hom : walking_parallel_pair → walking_parallel_pair → Type v
+@[derive _root_.decidable_eq] inductive walking_parallel_pair_hom :
```

Comment:
Why is `_root_` needed here?

Reply:
After `open walking_parallel_pair`, `decidable_eq` refers to `category_theory.limits.walking_parallel_pair.decidable_eq` (i.e., the `decidable_eq` instance derived for `walking_parallel_pair`).

Reply:
Discussed [here](https://leanprover.zulipchat.com/#narrow/stream/113488-general/topic/derive.20handler.20for.20fintype), and a fix is in the works, so we'll leave this as is for now.